node/bin/src/config/mod.rs

```diff
@@ -44,6 +44,8 @@ pub struct Config {
     pub batch_verification_config: BatchVerificationConfig,
 }
 
+pub const DEFAULT_ROCKS_DB_PATH: &str = "./db/node1";
```

Comment:
There exists the [config_constants](https://github.com/matter-labs/zksync-os-server/blob/main/node/bin/src/config_constants.rs) module; it probably makes sense to put this constant there. Even though it was introduced for constants that can change from time to time, I guess it still makes sense to keep things consistent.